src/app/components/FooterList/index.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import styled from 'styled-components';
+import { string, arrayOf, shape } from 'prop-types';
+import { C_WHITE, GEL_SPACING } from '../../lib/constants/styles';
+import Link from '../Link';
+
+const StyledList = styled.ul`
+  border-bottom: 1px solid ${C_WHITE};
+  display: grid;
+  grid-auto-flow: column;
+  grid-template-columns: repeat(2, 50%);
+  grid-template-rows: repeat(4, 50%);
+  list-style-type: none;
+  margin: 0;
+  padding: ${GEL_SPACING}px 0;
```

Comment:
Following the merging on https://github.com/bbc/simorgh/pull/475 `${GEL_SPACING}px` should be `${GEL_SPACING}` as the constant variable is now applied in rem's instead of a number with `px` appended